### ISSUE TITLE
Montar entorno de testing de UI en espacio compartido

### ISSUE THREAD

**Issue by jgmel** [phicus/krill#75]

**Comment by jgmel** [comment]
Validamos demokrill como servidor para pasar los test por CLI

**Comment by VictorGuerreroo** [comment]
@jgmel Yo confío en tu criterio

**Comment by jgmel** [comment]
Y el criterio de @paveldelpozo confirma?

**Comment by jgmel** [comment]
@VictorGuerreroo puedes hacer la comparativa de los tiempos que tarda `adt` y `dem` en pasar los test desde la consola?

**Comment by jgmel** [comment]
De momento fallan, los test siempre tiran de demokrill.phicus.es y actualmente la version 1.0.0 de icingadb nos está causando problemas

**Comment by jgmel** [comment]
Aplazamos esta tarea hasta tener disponible https://github.com/phicus/krill/issues/85, ya que va a ser el servidor destinado ha hacer los test

**Comment by jgmel** [comment]
Ya tenemos el entorno para montar los test

**Comment by paveldelpozo** [comment]
`krill-pre` tiene servidor gráfico? Para poder lanzar en él los test con previsualización?

**Comment by jgmel** [comment]
No, seria necesario? puedes hacer forwarding de X y usar tu pantalla, siempre va a ser mejor que cualquier VNC

**Comment by jgmel** [comment]
Yo creo que es más óptimo, siempre y cuando se pueda visualizar un resumen

**Comment by jgmel** [comment]
- [x]  Cambiar Nombre OLTS, oltHuawei
- [ ]  50 cpes de palo x realm
- [x]  BRIGE O ROUTER = graficas = NO TOCAR
- [ ]  Provision temporal - ont bridge
- [x]  Installer tools
- [x]  Copia de rubik / mockup - cobertura de red

**Comment by jgmel** [comment]
Me falta la ONT bridge por problemas logisticos. ¿Ssigue siendo necesaria?
